zephyr/wrapper.c

```diff
@@ -551,7 +553,7 @@ static inline const void *smex_placeholder_f(void)
 /* Need to actually use the function and export something otherwise the
  * compiler optimizes everything away.
  */
-const void *_smex_placeholder;
+static const void *_smex_placeholder;
```

Comment:
Sorry I haven't looked at the whole thing yet but I'm afraid this static contradicts its own comment again...